cmd/sriovdp/main.go

```diff
@@ -33,6 +33,10 @@ func flagInit(cp *cliParams) {
 		"JSON device pool config file location")
 	flag.StringVar(&cp.resourcePrefix, "resource-prefix", "intel.com",
 		"resource name prefix used for K8s extended resource")
+	flag.BoolVar(&cp.useCdi, "use-cdi", true,
+		"Use Container Device Interface to expose devices in containers")
+	flag.BoolVar(&cp.systemd, "systemd", false,
```

Comment:
do you think adding the systemd service to sriov-device-plugin is better that add it to SR-IOV operator?

Reply:
Since we're implementing CDI support as a part of device plugin project, I think it's reasonable it have systemd service implementation in the same place. SR-IOV Operatir will be responsible for service provisioning

Reply:
WDYT about having a separate binary for it under cmd ?

also the flag name systemd is confusing.
what this does is: it runs once to re-generate CDI configuration.

Reply:
no systemd is needed